.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,43 @@
+name: "CS"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  coding-standards:
+    name: "CS Fixer & PHPStan"
+    runs-on: "ubuntu-20.04"
+
+    env:
+      SYMFONY_PHPUNIT_VERSION: 7.5
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "7.4"
+          tools: cs2pr, composer:v1
```

Comment:
What is `cs2pr` for?

Reply:
Instead of dumping error inside the build, cs2pr will inline comment in the _files changed_ tab.
See that example 
![image](https://user-images.githubusercontent.com/62333/102337623-00bc1280-3f93-11eb-88ff-3a596c95e844.png)

Reply:
Super cool!